metadata-models/src/main/pegasus/com/linkedin/dataquality/DataQualityDimensionInfo.pdl

```diff
@@ -0,0 +1,32 @@
+namespace com.linkedin.dataquality
+
+/**
+ * Record to capture Dataset/Field level Data Quality Metrics
+ */
+record DataQualityDimensionInfo {
+
+  /**
+  * dimensions attribute to capture dimensions. It is an array of dimensionScore
+  * Required if this record is initialized
+  */
+  dimensions: array[DimensionScore]
+
+ /**
+  * tool_name attribute to capture the tool that has been used to generate the quality metrics.
+  * Optional field
+  */
+  tool_name: optional string
```

Comment:
Shouldn't we use camel case here as well?

Reply:
We should

Reply:
Updated to camelCase 'toolName:'